modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/dependencies/multi-task-workflow-definition.xml

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0"?>
```

Comment:
@davisaints can you remind me the need for a new xml?

Reply:
CanProgressToNextStepWhenParallelTasksCompleted test is using. We have a similar xml, but it is in the dependencies of the workflow metrics tests.